src/components/form/form.style.js

```diff
@@ -64,6 +67,8 @@ const formBottomMargins = (fieldSpacing) =>
     7: "var(--spacing700)",
   }[fieldSpacing]);
 
+const HEIGHT_SPACING = 216;
```

Comment:
I agree - but I don't see a better way of doing it, if Ed's tested it with various different size variants, on different screen sizes and it works OK then it's probably fine.

I was concerned that on a mobile in landscape orientation, this could cause the height to be too small, particularly if there was no header - but I just tested that and it still seems OK (and with a surprisingly large amount of content visible even with both a sticky header and footer), so I think this approach, ugly as it is, is probably OK at least to start with,

Reply:
Yeah I've tried everything else I can to avoid this and found either they don't work or they are less than optimal for what we are trying to achieve